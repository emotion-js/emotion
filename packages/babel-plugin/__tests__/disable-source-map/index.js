import babelTester from 'babel-tester'
import plugin from '@emotion/babel-plugin'

babelTester('disable source map', __dirname, {
  plugins: [[plugin, { sourceMap: false }]]
})
