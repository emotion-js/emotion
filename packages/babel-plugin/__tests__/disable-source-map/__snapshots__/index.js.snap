// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`disable source map css 1`] = `
"import { css } from '@emotion/react'

let style = css({ color: 'hotpink' })


      ↓ ↓ ↓ ↓ ↓ ↓

import { css } from '@emotion/react';
let style = {
  name: \\"3sn2xs\\",
  styles: \\"color:hotpink\\"
};"
`;

exports[`disable source map styled 1`] = `
"import styled from '@emotion/styled'

let Comp = styled.div({ color: 'hotpink' })


      ↓ ↓ ↓ ↓ ↓ ↓

import _styled from \\"@emotion/styled/base\\";

let Comp = _styled(\\"div\\", {
  target: \\"eggnygh0\\",
  label: \\"Comp\\"
})({
  name: \\"3sn2xs\\",
  styles: \\"color:hotpink\\"
});"
`;

exports[`disable source map vanilla 1`] = `
"import { css } from '@emotion/css'

let cls = css({ color: 'hotpink' })


      ↓ ↓ ↓ ↓ ↓ ↓

import { css } from '@emotion/css';
let cls =
/*#__PURE__*/
css({
  name: \\"3sn2xs\\",
  styles: \\"color:hotpink\\"
});"
`;
